map:
    topic_name: /map
    enable_erosion: true
    erosion_gap: 3
path_publisher:
    enable: true
    topic_name: /path
    frame_id: map
planner_service_name: /get_path
pose_tracker:
    stamped: true
    topic_name: /tracked_pose
obstacle_occupancy_threshold: 3
goal_distance_threshold: 0.5
trajectory_optimizer:
    repulsion_gain: 50.0
    potential_field_radius: 12 # Robot's medium size multiplied by resolution is recommended
